src/Utils/GalleryUtils.ts

```diff
@@ -281,6 +286,17 @@ export function downloadItem(
   );
 }
 
+export const removeNotebookViewerLink = (notebook: Notebook, newCellId: string): void => {
+  if (!newCellId) {
+    return;
+  }
+  const notebookV4 = notebook as NotebookV4;
+  if (notebookV4 && notebookV4.cells[0].source[0].search(newCellId)) {
```

Comment:
Consider using optional chaining in case `cells` or `source` length is < 1.
```suggestion
  if (notebookV4?.cells[0]?.source[0]?.search(newCellId)) {
```
This works, but consider using nteract's selectors to access and update the inside of the notebook model so that we don't have worry if the format changes, etc. (I know you can find a cell, and probably delete one at least).